Homework_Week_4/task_3_testing/atm_tests.py

```diff
@@ -0,0 +1,50 @@
+import unittest
```

Comment:
⚠️ (-3) well done on completing the unit tests. You managed to test the base case.
Would be good if you could add another test to test the edge cases i.e.
- test case where failed to input more than 3 times
- where user input less than 4 digits
- when user tries to withdraw more than the balance